simulator/src/css/testCreator.css

```diff
@@ -139,4 +139,15 @@
 
 body {
     overflow: scroll;
-}
\ No newline at end of file
+}
+
+.right-button-group .lower-button:hover
+{
+    background-color: #42b983;
+    border-radius: 4px;
+    color: #fff;
```

Comment:
1. Instead of using color HEX code, you can use the defined variables.
2. Try to check if it's possible to use the same class name used for primary buttons in CircuitVerse

Reply:
Since it is a regular css file use of anything like ($primary-green) wont work.
So I will modify it a bit differently, and take the color code to the :root block as you can check the commits.